src/axom/primal/operators/in_sphere.hpp

```diff
@@ -73,19 +97,39 @@ inline bool in_sphere(const Point<T, 3>& q,
                       const Point<T, 3>& p0,
                       const Point<T, 3>& p1,
                       const Point<T, 3>& p2,
-                      const Point<T, 3>& p3)
+                      const Point<T, 3>& p3,
+                      double EPS = 1e-8)
```

Comment:
This change significantly improves the efficiency of `in_sphere()` in 3D by converting it from a 5x5 determinant to a 4x4 determinant.

We have an explicit implementation of 4x4 determinants, but 5x5 determinants are based on a general LU decomposition.